internal/tensorzero-node/lib/bindings/RunEvaluationRequest.ts

```diff
@@ -0,0 +1,62 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+import type { EvaluationConfig } from "./EvaluationConfig";
+import type { EvaluationFunctionConfig } from "./EvaluationFunctionConfig";
+import type { UninitializedVariantInfo } from "./UninitializedVariantInfo";
+
+/**
+ * Request body for running an evaluation.
+ */
+export type RunEvaluationRequest = {
+  /**
+   * URL of the gateway to use for inference requests
+   */
+  gateway_url: string;
+  /**
+   * URL of the ClickHouse database
+   */
+  clickhouse_url: string;
```

Comment:
If you call the gateway, it should be itself? Remove these two and use the gateway you're calling (and whatever CH it's using).

Reply:
yeah good catch, didn't read this closely.